src/doc/rustc/target_infos/*-apple-tvos.md

```diff
@@ -0,0 +1,78 @@
+---
```

Comment:
I think it makes sense to put a very big warning comment in the old docs for the docs that have already been migrated, reminding people to update them in both places to keep them in sync for now.